<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulePage</class>
 <widget class="QWidget" name="SchedulePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QComboBox" name="toTownComboBox"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="searchPushButton">
     <property name="text">
      <string>поиск</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="fromTownComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableView" name="scheduleTableView"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="allPushButton">
     <property name="text">
      <string>все</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
